### PR TITLE
ISSUE-163 adding default tags and resource tags

### PR DESCRIPTION
**Pull request by sandipnahak**
### Description

Thanks for contributing this Pull Request. Make sure that you submit this Pull Request against the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues : (compulsory)
https://github.com/intuit/Trapheus/issues/163
- Relevant PRs : (optional)
- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [x] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

This will add tags at cfn stack level as well as the resource level.

### CODE REVIEW

**Comment by sandipnahak** [comment]
#165

**Comment by sandipnahak** [comment]
@stationeros Please review

**Comment by stationeros** [comment]
Sandip,

I will be able to get to this by Tuesday as I am out for a visa
appointments. Let me know if that works

File: README.md [hunk 1]
```diff
@@ -95,6 +95,25 @@ The following are the parameters for creating the cloudformation template:
 
 [![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/colored.png)](#instructions)
 
+## Tagging
```
Comment (stationeros) [inline README.md:98]: Can this be a seperate markdown file under docs . The worry is that the readme is just increasing in size

File: samconfig.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+version = 0.1
+
+[default.deploy.parameters]
```
Comment (stationeros) [inline samconfig.toml:3]: Have we tested this ?

Comment (sandipnahak) [reply samconfig.toml:3]: yes, I could create the stack using these params
Here is the AWS ref doc https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html

File: README.md (hunk 1 above)
Comment (sandipnahak) [reply README.md:98]: sure

**Review by stationeros — APPROVED** [review]

**Pull request merged**